modular_chomp/code/game/jobs/job/noncrew.dm

```diff
@@ -65,7 +65,6 @@
 		return FALSE
 
 /datum/job/shadekin/update_limit(var/comperator)
-	var/current_limit = clamp(comperator / 5, 3, 5) + round(comperator / 50)
```

Comment:
no... I said the proc explodes if you remove it... comment the entire section out

Reply:
Ackschually
![image](https://github.com/CHOMPStation2/CHOMPStation2/assets/15962836/737d40d7-8fdc-4e69-8896-5c90a6ea47c3)
![image](https://github.com/CHOMPStation2/CHOMPStation2/assets/15962836/737d40d7-8fdc-4e69-8896-5c90a6ea47c3)